src/service/api.tsx

```diff
@@ -107,26 +107,30 @@ const api = {
     return obj.data;
   },
 
-  async getPopularQueryList() {
-    const obj = await axios.get(`movie/popular?${API_KEY}&query`);
+  async getPopularQueryList(page: number = 1) {
+    const obj = await axios.get(`movie/popular?${API_KEY}&query&page=${page}`);
     return obj.data.results;
   },
 
-  async getTopRatedList() {
-    const obj = await axios.get(
-      `movie/top_rated?translations&${API_KEY}&region=US`
-    );
+  async getTopRatedList(page: number = 1) {
+    const obj = await axios.get(`movie/top_rated?translations&${API_KEY}&region=US&page=${page}`);
     return obj.data.results;
   },
 
-  async getSearchList(query: string) {
-    let obj = await axios.get(`search/movie?${API_KEY}&query=${query}`);
+  async getSearchList(query: string, page: number = 1) {
+    let obj = await axios.get(`search/movie?${API_KEY}&query=${query}&page=${page}`);
     return obj.data.results;
   },
 
-  changeImgLinks(url: string, size: string) {
+  changeImgLinks(url: string, size: string = 'w500') {
     return `${API_IMG_URL}${size}${url}`;
-  }
-};
+  },
+
+  changeListByPagination(arr: Array<{}>, page: number = 1) {
```

Comment:
where is return type? Array<{}> - is wrong... here you might use generics - https://www.typescriptlang.org/docs/handbook/2/generics.html
`changeListByPagination(arr: Array<T>, page: number = 1): Array<T> {`

Reply:
have you seen this comment @mrSelieznov it is as easy as ctrl+c > ctrl+v as solution already here